starter-code/src/functions-and-arrays.js

```diff
@@ -11,13 +22,65 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(array){
+  for (var longestWord = '', i = 0; i < array.length; i++) {
+    if (array[i].length > longestWord.length) {
+      longestWord = array[i];
+    }    
+  }
+  return longestWord;
+}
```

Comment:
No inicialices una variable dentro del for si no la vas a usar para nada relacionado con el for, ya que da poca claridad. Mejor creala e inicialízala fuera.  Por lo demás es correcto